fulp_modules/features/antag_tips/html/bloodsucker.html

```diff
@@ -10,6 +10,7 @@ <h1>You are a Bloodsucker!</h1>
 	<p>Once you have a Coffin claimed, you will learn new structures in the <b>Tribal</b> section of your crafting menu.</p>
 	<p><img src="vassalrack.png"/>Examine your new structures to see how they work to your advantage.</p>
 	<p>Beware: Other Bloodsuckers are not necessarily your friends, but your survival may depend on cooperation. Betray them at your own discretion and peril.</p>
+	<p><b>NOTE: If you fireman carry someone, they cannot be stuffed into a Coffin without buckling them onto a chair first. (Yes, this is a bug, hopefully it will be fixed soon.)</b></p>
```

Comment:
the absolute state

Reply:
I have a PR up on TG to fix it, just waiting for it to get re-reviewed mostly.